server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -1423,11 +1426,11 @@ public void testFullSnapshotUnassignedShards() {
                 }
             })
 
-            // Take a full snapshot for use as the source for future clones
+            // Take the snapshot to check the reaction to having unassigned shards
             .<Void>andThen(
                 (l, ignored) -> client().admin()
                     .cluster()
-                    .prepareCreateSnapshot(repoName, originalSnapshotName)
+                    .prepareCreateSnapshot(repoName, "test-snapshot")
```

Comment:
where is this name coming from? Could we make it more clear with a comment, variable name or declaration, or something?